src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -23,16 +22,14 @@
 using Microsoft.VisualStudio.R.Package.Repl;
 
 namespace Microsoft.R.Components.InteractiveWorkflow.Implementation {
-    public sealed class RInteractiveWorkflow : IRInteractiveWorkflow {
+    public sealed class RInteractiveWorkflow : IRInteractiveWorkflowVisual {
```

Comment:
But we don't have to define `IInteractiveWindowVisualComponent` in xplat. `ActiveWindow` is used only in `RHistory` (which we won't convert into xplat right now) and in one place in RPlotManager, where it can be replaced with `IUIService.UpdateCommandStatus`. The rest are either Interactive Window bits (they can't be xplat), or code in Package project.